assets/js/state/sagas/notifications.jsx

```diff
@@ -2,16 +2,40 @@
 import React from 'react';
 import { takeEvery } from 'redux-saga/effects';
 import { toast } from 'react-hot-toast';
-import { NOTIFICATION } from '@state/notifications';
+import {
+  NOTIFICATION,
+  DISMISSABLE_NOTIFICATION,
+  DISMISS_NOTIFICATION,
+} from '@state/notifications';
+import DismissableToast from '@common/DismissableToast';
 
 export function* notification({ payload }) {
-  const { text, icon } = payload;
+  const { text, icon, id, duration } = payload;
```

Comment:
call me crazy, but normally, the `id` is the first variable to deconstruct :see_no_evil:

Reply:
done